_data/speakers.yml

```diff
@@ -428,17 +445,18 @@
   image: osthege_06.jpg
   thumbnail: osthege_06.jpg
   alt: Michael
-  topic:
-  proposal_summary:
+  topic: calibr8: Going beyond linear ranges with non-linear calibration curves and multilevel modeling
+  proposal_summary: You just coded up a beautiful model and dummy prediction looks great. Now comes the data, but wait: the units don't match! And to make matters worse, the correlation between model variable and measurement readout is non-linear and heteroscedastic! Sounds familiar? non-linear calibration to the rescue! With `calibr8`, we present a statistical framework and corresponding open source Python package that solves non-linear calibration and likelihood functions for modeling. From a laboratory automation and systems biology perspective, the advantage of non-linear calibration with `calibr8` is two-fold: For lab scientists doing (high-throughput) experiments, `calibr8` facilitates more intuitive uncertainty quantification and makes every-day data analysis more robust, automatable and Bayesian. From a modeling perspective, non-linear _error models_ are essential components of realistic Bayesian process models, and are key to accurately describe the nastiest step of the data-generating process. In this talk, we will take you step-by-step through the data-generating process of an automated bioassay and demonstrate how its non-linearities are modeled with `calibr8`. We will show how `calibr8` can make your life easier - and of course more Bayesian - even if you don't always go all the way to a process model. Finally, we will show how non-linear error models and multi-level modeling with `PyMC3` enable you to get more information out of heterogeneous regression analyses. Join us for the talk and discussion to learn about building Bayesian models for bioassays and dive into the fascinatingly frightening world of non-linear measurement errors!
   discourse_link:
   youtube_link:
   description: Michael Osthege is a biotech Bayesian by choice.
     He likes to work with robots, bacteria and models as much as he loves to work in enthusiastic teams.
     As a PhD student in laboratory automation for bioprocess development at Forschungszentrum Jülich, he writes software to make robots generate his data.
     Since he unit-tests his code, he always blames the robots if the data doesn't agree with his Bayesian models.
   presentation_type: Talk
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Micege
```

Comment:
Michael should have the same keyword as Laura, the first and longer comment above.

Reply:
Changed the keywork